### PR TITLE
chore(indexer): Disable flaky tests 

### PR DESCRIPTION
**Pull request by roninjin10**
https://app.circleci.com/insights/github/ethereum-optimism/optimism/workflows/main/tests?branch=develop

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #7930** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7930" target="_blank">Graphite</a> 
    * **PR #7929** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7929" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7929?utm_source=stack-comment).

**Review by ajsutton — APPROVED** [review]
Approving but I'm nervous about this as it opens a risk that we'll break stuff in indexer because of go.mod updates or things like that without noticing.

Can you make sure we have an issue logged to re-enable this so we don't forget? Probably worth doing this only for a limited time (eg 2 weeks) and setting a reminder about it in slack or somewhere too.

File: ops/check-changed/main.py [hunk 1]
```diff
@@ -56,7 +56,11 @@
 
 def main():
     patterns = sys.argv[1].split(',')
-    patterns = patterns + REBUILD_ALL_PATTERNS
+
```
Comment (hamdiallam) [inline ops/check-changed/main.py:59]: holding open

**Comment by hamdiallam** [comment]
> Approving but I'm nervous about this as it opens a risk that we'll break stuff in indexer because of go.mod updates or things like that without noticing.

+1, there's some PRs in the works to help better manage lifecycle of resource between the indexer and op-e2e unit tests. Lets see if those fix these issues before merging this in

**Comment by roninjin10** [comment]
Guessing we are good to close this but can reopen if I'm mistaken

**Pull request closed without merging**